### DISCUSSION TITLE
Only month and year selection

### DISCUSSION THREAD

**Discussion by santanapaulo** [gpbl/react-day-picker#2505] [Support] [answered]
I am wondering if there is a possibility to use the date-picker only to select month and year...

In my case, I don't need to know the day. I just have to get month and year. In that case, will be ackward to open the dropdown and see disabled days. I want to NOT display the dropdown day.

**Comment by gpbl** [comment, marked as answer]
Hi @santanapaulo, you can use `onMonthChange` and the `month` prop to control the month and keep it unselectable. You can also change the dropdown with a custom component via the `components` prop. Have you tried this solution?

Consider also that a _day_ picker may not be the good choice for a month / year selection.

**Reply by santanapaulo** [reply]
Thanks for the reply!!

I did not try the solution you mention, but I will.
And i've search some solutions online but I did not like it, so that's why I wanna keep RDP (event thought I know it's a _day_ picker).
